build.sbt

```diff
@@ -29,26 +27,32 @@ libraryDependencies ++= Seq(
 
 ) ++ Seq("ssm", "s3", "url-connection-client").map(artifact => "software.amazon.awssdk" % artifact % "2.20.95")
 
-enablePlugins(RiffRaffArtifact, BuildInfoPlugin)
-
-assemblyJarName := s"${name.value}.jar"
-riffRaffPackageType := assembly.value
-riffRaffUploadArtifactBucket := Option("riffraff-artifact")
-riffRaffUploadManifestBucket := Option("riffraff-builds")
-riffRaffArtifactResources += (file("cdk/cdk.out/GeoipDbRefresher-PROD.template.json"), "cloudformation/cfn.json")
+enablePlugins(BuildInfoPlugin)
 
+assembly / assemblyOutputPath  := file(s"target/${name.value}.jar")
```

Comment:
Ah! TIL! We've been [`mv`ing files around](https://github.com/guardian/security-hq/blob/b1e63bde3529267697c699626e1f4af2ca45232e/script/ci#L12-L14) 😅